apps/prairielearn/src/tests/groupExam.test.js

```diff
@@ -0,0 +1,401 @@
+const { config } = require('../lib/config');
+const assert = require('chai').assert;
+const cheerio = require('cheerio');
+const fetch = require('node-fetch').default;
+const { step } = require('mocha-steps');
+
+const sqldb = require('@prairielearn/postgres');
+const sql = sqldb.loadSqlEquiv(__filename);
+
+const helperServer = require('./helperServer');
+const { idsEqual } = require('../lib/id');
+const { TEST_COURSE_PATH } = require('../lib/paths');
+
+let page, elemList;
+const locals = {};
+locals.helperClient = require('./helperClient');
```

Comment:
What is this doing? I can't find anywhere that actually reads this.

Reply:
You're correct, it's doing nothing 😅 it was carried over from the boilerplate of `groupStudent.test.js`; I'll remove it